TextArea: add a note about input (#6507)
DevExpress/devextreme-documentation: concepts/05 UI Components/TextArea/13 Handle the Keyboard Events.md

> The TextArea raises three keyboard events: [keyDown](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyDown.md '/Documentation/ApiReference/UI_Components/dxTextArea/Events/#keyDown'), [keyUp](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyUp.md '/Documentation/ApiReference/UI_Components/dxTextArea/Events/#keyUp') and [enterKey](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/enterKey.md '/Documentation/ApiReference/UI_Components/dxTextArea/Events/#enterKey'). Within the functions that handle them, you can access the original keyboard events. If you are _not_ going to change the functions during the lifetime of the UI component, assign them to the respective UI component properties.

Reviewer: ```suggestion
The TextArea raises three keyboard events: [keyDown](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyDown.md '/Documentation/ApiReference/UI_Components/dxTextArea/Events/#keyDown'), [keyUp](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyUp.md '/Documentation/ApiReference/UI_Components/dxTextArea/Events/#keyUp') and [enterKey](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/enterKey.md '/Documentation/ApiReference/UI_Components/dxTextArea/Events/#enterKey'). Within functions that handle them, you can access original keyboard events. If you are _not_ going to change functions during the lifetime of the UI component, assign them to respective UI component properties.
```

---

> [note] Another event that you can use is [input](/Documentation/ApiReference/UI_Components/dxTextArea/Events/#input). This is not a strictly keyboard event, a mouse action can also change TextArea input value.

Reviewer: ```suggestion
[note] You can also use  the [input](/Documentation/ApiReference/UI_Components/dxTextArea/Events/#input) event. While not a strictly keyboard event, a mouse action can also change a TextArea input value.
```
